config/routes.rb

```diff
@@ -22,5 +22,9 @@
   get '*path', to: 'home#app'
   get '/*undefined', to: redirect('/')
 
+  namespace :admin do
+    resources :teams
+  end
```

Comment:
On line 4 of routes.rb:
```
ActiveAdmin.routes(self) rescue ActiveAdmin::DatabaseHitDuringLoad
```
This inserts all of your ActiveAdmin routes, including this (you can see it  on lines 69-80 by doing `$ rails routes`:

```
             admin_teams GET        /admin/teams(.:format)                                                                            admin/teams#index
                         POST       /admin/teams(.:format)                                                                            admin/teams#create
          new_admin_team GET        /admin/teams/new(.:format)                                                                        admin/teams#new
         edit_admin_team GET        /admin/teams/:id/edit(.:format)                                                                   admin/teams#edit
              admin_team GET        /admin/teams/:id(.:format)                                                                        admin/teams#show
                         PATCH      /admin/teams/:id(.:format)                                                                        admin/teams#update
                         PUT        /admin/teams/:id(.:format)                                                                        admin/teams#update
                         DELETE     /admin/teams/:id(.:format)                                                                        admin/teams#destroy
```

What you have here is repeating the same thing, on lines 144-151:
```
GET        /admin/teams(.:format)                                                                            admin/teams#index
POST       /admin/teams(.:format)                                                                            admin/teams#create
GET        /admin/teams/new(.:format)                                                                        admin/teams#new
GET        /admin/teams/:id/edit(.:format)                                                                   admin/teams#edit
GET        /admin/teams/:id(.:format)                                                                        admin/teams#show
PATCH      /admin/teams/:id(.:format)                                                                        admin/teams#update
PUT        /admin/teams/:id(.:format)                                                                        admin/teams#update
DELETE     /admin/teams/:id(.:format)    
```
Can you try deleting lines 25-27 and see if it still works?

Reply:
sorry, I use ActiveAdmin.routes(self) rescue ActiveAdmin::DatabaseHitDuringLoad